src/firebase/rtdb.js

```diff
@@ -0,0 +1,3 @@
+import app from "./app.js";
+import { getDatabase } from "firebase/database";
+export default getDatabase(app);
```

Comment:
I guess we did break out some aspects of importing the RTDB

Not sure what this does exactly that is different from the stuff at the top of players_in_room.js